test_setup.alerts_logger.yml

```diff
@@ -0,0 +1,41 @@
+---
+# ========================================
+#  QE configuration for alerts logger
+# ========================================
+#
+# This playbook serves as setup of alerts logger on tendrl server machine.
+# It periodically logs alerts from tendrl api based on tendrl user.
+
+- hosts: usm_server
```

Comment:
I would rather have this installed on a client machine, as the snmp and smtp alerting logs are stored there, so that we will have all alerting logs on the same machine. Moreover it would keep our usm server pristine.